kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -274,12 +271,16 @@ def _is_known_ci_env(known_ci_env_var_keys: set[str]):
     return any(os.getenv(key) for key in known_ci_env_var_keys)
 
 
-def _get_project_properties(user_uuid: str, pyproject_path: Path) -> dict:
-    project_id = _get_or_create_project_id(pyproject_path)
-    package_name = PACKAGE_NAME or UNDEFINED_PACKAGE_NAME
-    hashed_project_id = (
-        _hash(f"{project_id}{package_name}") if project_id is not None else None
-    )
+def _get_project_properties(user_uuid: str, project_path: Path) -> dict:
+    if project_path:
+        pyproject_path = project_path / PYPROJECT_CONFIG_NAME
```

Comment:
Instead of overwriting the supplied `project_path` here I'd just create a new variable to make it easier to follow what happens and debug if something is wrong.

Reply:
I believe it's already a new variable?